### PR TITLE
T3chguy/hide level change on 50

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/1543

### CODE REVIEW

**Comment by turt2live** [comment]
There's a ton of formatting changes here, and I can't actually see what the change is.

**Comment by t3chguy** [comment]
@turt2live they're in two separate commits :)

**Comment by turt2live** [comment]
imo, the delint is something that shouldn't be part of this, but that's not my call.

**Comment by t3chguy** [comment]
I can back it out if someone else agrees

File: src/components/views/rooms/MemberInfo.js [hunk 1]
```diff
@@ -788,11 +779,15 @@ module.exports = withMatrixClient(React.createClass({
...
+        let presenceCurrentlyActive;
+
         if (this.props.member.user) {
-            var presenceState = this.props.member.user.presence;
-            var presenceLastActiveAgo = this.props.member.user.lastActiveAgo;
-            const presenceLastTs = this.props.member.user.lastPresenceTs;
-            var presenceCurrentlyActive = this.props.member.user.currentlyActive;
+            presenceState = this.props.member.user.presence;
+            presenceLastActiveAgo = this.props.member.user.lastActiveAgo;
+            // const presenceLastTs = this.props.member.user.lastPresenceTs;
```
Comment (lukebarnard1) [inline src/components/views/rooms/MemberInfo.js]: I would just get rid of this if it's not being used

**Comment by lukebarnard1** [comment]
So long as the delint is in a separate commit, I think it's fine. And thanks for doing it!

LGTM apart from small comment.

**Comment by t3chguy** [comment]
@lukebarnard1 done! :)

**Pull request merged**